### PR TITLE
Feat/convert to vite

### PR DESCRIPTION
**Pull request by b54690**
## What's new?

- Added Vite config to Playground and Storybook modules.
- Removed Webpack in it's entirety. 
- Updated node version to 16.14.2 to work with Storybook Vite plugin 

## Ticket number(s) in JIRA (if internal)

ARM-205

[board](https://rocketmakers.atlassian.net/jira/software/projects/HM/boards/172)

### CODE REVIEW

File: module/.storybook/main.js [hunk 1]
```diff
@@ -1,59 +1,27 @@
...
-      use: [
-        'style-loader',
-        {
-          loader: 'css-loader', // turn url() and @import calls into require
-          options: {
-            sourceMap: isDevelopment,
-          },
-        },
-        {
-          loader: 'postcss-loader',
```
Comment (judithrocketmakers) [inline module/.storybook/main.js:26]: Given we no longer have this config, check that we still need all the dependencies in the package if you haven't already?

**Review by judithrocketmakers — APPROVED** [review]
Think this all looks okay, one comment that I suspect you've already done.

**Comment by judithrocketmakers** [comment]
Hmm. I've just tried building this locally as the docs site is down and I get a stack of errors when I try...

**Comment by judithrocketmakers** [comment]
Okay, so looks like Cypress isn't compatible with the npm upgrade - needs a --legacy=peer-deps or to be stripped out. Don't know if it's easier to merge in my branch first at that point.

**Review by adamiprinciples — COMMENTED** [review]
I think we need to clean up the package.json a bit before this goes in. There are a lot of redundant script steps and dependencies that can go

**Comment by judithrocketmakers** [comment]
Also need to double check this all builds as expected, given dev currently doesn't; the culprit is Cypress which is coming out in my branch so I'm no longer sure which is the sensible way around to merge these.

**Comment by b54690** [comment]
Closing PR as solution need to be revised.

**Comment by judithrocketmakers** [comment]
As Vite is up and running on the rewrite branch, deleting this branch for clarity.

**Pull request closed without merging**